libs/coin-modules/coin-stacks/src/bridge/utils/misc.unit.test.ts

```diff
@@ -1,7 +1,7 @@
 import { mapPendingTxToOps, mapTxToOps } from "./misc";
-import { encodeAccountId } from "../../../../account";
-import { TransactionResponse } from "./api.types";
-import { fetchFullTxs } from "./api";
+import { encodeAccountId } from "@ledgerhq/coin-framework/account/index";
+import { TransactionResponse } from "../../network/api.types";
+import { fetchFullTxs } from "../../network/api";
```

Comment:
Perhaps you can have some `index.ts` in `network` ?

Reply:
Done in c726e50ef9396e8552e7f89fe4e5652addbf6084